hack/run-ci-e2e-test.sh

```diff
@@ -169,10 +169,6 @@ if [[ "$TEST" = "all" || "$TEST" = "upgrade" ]]; then
   go test ./test/e2e/... -run=TestWMCO/upgrade -v -timeout=90m -args $GO_TEST_ARGS
 
   # Run the reconfiguration test
-  # The reconfiguration suite must be run directly before the deletion suite. This is because we do not
```

Comment:
Isn't this still true?

Reply:
No it is not true anymore. This change is making it so the test waits nodes to fully reconcile after changing the private key back to the valid key.